### DISCUSSION TITLE
Cannot use "Subsetdata" and “ident.remove” in version 4.0.2

### DISCUSSION THREAD

**Discussion by DY-dy2021** [satijalab/seurat#4559] [Q&A] [answered]
Hello everyone,

I upgrade Seurat 3.2.2 to 4.0.2, but I cannot use the "Subsetdata" and “ident.remove”, and I cannot find the substitute of these command. So, how can I use similar command like "Subsetdata" in the latest version Seurat？

I would appreciate if anyone may help me.

Thanks!

**Comment by samuel-marsh** [comment, marked as answer]
Hi,

Please see `subset` function.  You can use `invert = TRUE` to remove clusters instead of keeping them.

Best,
Sam

**Reply by DY-dy2021** [reply]
Oh thanks! I have already found in the Vignettes! 

Best,
DY
